.github/workflows/sanitizer-tests.yaml

```diff
@@ -0,0 +1,190 @@
+name: Sanitizer analysis
+
+on:
+  workflow_dispatch:
+    inputs:
+      discovery_server_branch:
+        description: 'Branch or tag of Discovery Server repository (https://github.com/eProsima/Discovery-Server)'
+        required: true
+        default: 'v1.2.1'
+      fastdds_branch:
+        description: 'Branch or tag of Fast DDS repository (https://github.com/eProsima/Fast-DDS)'
+        required: true
+        default: 'master'
```

Comment:
Same as comment here

Reply:
dcc3314

Reply:
I think it is still not applied, neither in lines 17, 31  and 115 of this same workflow

Reply:
7f5610dfb